modules/sdk-coin-avaxp/src/lib/iface.ts

```diff
@@ -2,6 +2,14 @@ import { Entry, TransactionExplanation as BaseTransactionExplanation, Transactio
 
 export interface TransactionExplanation extends BaseTransactionExplanation {
   type: TransactionType;
+  fromAddresses: string[];
+  rewardAddresses: string[];
```

Comment:
curious if we're duplicating information... how are `rewardAddresses` different from the addresses in `outputs`?

if we see that `type === staking`, does it mean that the addresses in `outputs` are reward addresses?